oe_corporate_site_info.module

```diff
@@ -6,3 +6,188 @@
+    ];
+    $form['oe_site_information']['content_owners'][$i]['_weight'] = [
+      '#type' => 'weight',
+      '#title' => t('Weight for row @number', ['@number' => $i + 1]),
+      '#title_display' => 'invisible',
+      '#delta' => $max,
+      '#default_value' => $content_owners[$i]['weight'] ?? 0,
+      '#weight' => 100,
+    ];
+  }
+  $form['oe_site_information']['content_owners'] += [
+    '#theme' => 'field_multiple_value_form',
+    '#field_name' => 'content_owners',
+    '#cardinality' => -1,
+    '#cardinality_multiple' => TRUE,
+    '#element_validate' => [
+      '_oe_corporate_site_info_content_owners_validate',
+    ],
+    '#required' => TRUE,
+    '#title' => t('Default content owner(s)'),
+    '#description' => t('This is not the writer of the content, but the subject matter expert responsible for keeping this content up to date. <br>When this field is populated, it will provide the default Content owner for all new content on this website. It can be overwritten for every new item.'),
+    '#max_delta' => ($max + 1),
+  ];
+  $id_prefix = 'content_owners';
+  $wrapper_id = Html::getUniqueId($id_prefix . '-add-more-wrapper');
+  $form['oe_site_information']['content_owners']['#prefix'] = '<div id="' . $wrapper_id . '">';
+  $form['oe_site_information']['content_owners']['#suffix'] = '</div>';
+  $form['oe_site_information']['content_owners']['add_more'] = [
+    '#type' => 'submit',
+    '#name' => strtr($id_prefix, '-', '_') . '_add_more',
+    '#value' => t('Add another item'),
+    '#attributes' => ['class' => ['field-add-more-submit']],
+    '#limit_validation_errors' => [['oe_site_information', 'content_owners'],],
+    '#submit' => ['_oe_corporate_site_info_add_more_submit'],
+    '#ajax' => [
+      'callback' => '_oe_corporate_site_info_add_more_ajax',
+      'wrapper' => $wrapper_id,
+      'effect' => 'fade',
+    ],
+  ];
+
+  $form['#submit'][] = '_oe_corporate_site_info_submit_config_form';
+}
+
+/**
+ * Validate callback for content owner fields.
+ */
+function _oe_corporate_site_info_content_owners_validate(&$element, FormStateInterface $form_state, &$complete_form): void {
+  $content_owners = $form_state->getValue('content_owners');
+  unset($content_owners['add_more']);
+  foreach($content_owners as $key => $content_owner) {
+    if (empty($content_owner['target'])) {
+      unset($content_owners[$key]);
+    }
+  }
+  if (empty($content_owners)) {
+    $form_state->setError($element, t('You have to select at least 1 content owner.'));
+  }
+}
+
+/**
+ * Submit handler for site information form.
+ */
+function _oe_corporate_site_info_submit_config_form(array $form, FormStateInterface $form_state): void {
+  $corporate_site_info = \Drupal::configFactory()->getEditable('oe_corporate_site_info.settings');
+  $site_owner = $form_state->getValue('site_owner', NULL);
+  // Save only valid site owner references.
+  if ($site_owner && \Drupal::entityTypeManager()->getStorage('skos_concept')->load($site_owner)) {
```

Comment:
THere is no real need for this, we can simply store the value coming from $form_state->getValue('site_owner', NULL);

Reply:
If we are worried about storing invalid references, should we not check that in the validation step?